lib/astarte/client/appengine/devices.ex

```diff
@@ -46,6 +46,48 @@ defmodule Astarte.Client.AppEngine.Devices do
     end
   end
 
+  def get_property_path_data(%AppEngine{} = client, device_id, interface, path, query_opts \\ [])
```

Comment:
What about merging this with the previous one by passing the path as opt (and moving query opts inside the opts) like this:
```
def get_properties_data(%AppEngine{} = client, device_id, interface, opts \\ []) do
  [...]
  request_path =
    if path = opts[:path] do
      "devices/#{device_id}/interfaces/#{interface}#{path}"
    else
      "devices/#{device_id}/interfaces/#{interface}
    end
  query = Keyword.get(opts, :query, [])
  ...
end

Reply:
Note that I expect the leading slash to be included in the path, so I don't add it after `interface`